bescort.lic

```diff
@@ -301,6 +302,39 @@ class Bescort
 
   private
 
+  def use_flying_mount(type, mode, speed = 'fly')
```

Comment:
What do you think about having two separate functions, one for mount and one for dismount?

Reply:
I started separating it and ended up with method names like this:
mount_flying_mount()
dismount_flying_mount()

They sounded strange so I'm going to leave it as one function for now.  I'm thinking about moving this whole method into common-travel.lic later so I'll address it then.